Detections/SecurityAlert/CorrelateIPC_Unfamiliar-Atypical

```diff
@@ -0,0 +1,46 @@
+id: a3df4a32-4805-4c6d-8699-f3c888af2f67
+name: Correlate Unfamiliar sign-in properties and atypical travel alerts
+description: |
+  'When a user has both an Unfamiliar sign-in properties alert and an Atypical travel alert within 20 minutes, the alert should be handled with a higher severity'
+severity: High
+requiredDataConnectors:
+  - connectorId: Azure Active Directory Identity Protection
+    dataTypes:
+      - Azure Active Directory Identity Protection
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
```

Comment:
Techniques should be completed, although not required.  As explained above, I would suggest - T1078

Reply:
Thanks 👍